agent/utils/ephemeral_ports_test.go

```diff
@@ -269,12 +267,20 @@ func TestVerifyPortsWithinRange(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.testName, func(t *testing.T) {
-			result := VerifyPortsWithinRange(tc.testActualRange, tc.testExpectedRange)
+			result := verifyPortsWithinRange(tc.testActualRange, tc.testExpectedRange)
 			assert.Equal(t, tc.expectedResult, result)
 		})
 	}
 }
 
+func TestResetTracker(t *testing.T) {
```

Comment:
This test `TestResetTracker()` has been reviewed in https://github.com/aws/amazon-ecs-agent/pull/3585.